CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Change Log
 
 
+## 1.0.0 - 2016-01-26
+
+### Removed
+
+- Stability config from composer
```

Comment:
Not to mention I have to revert it because integration tests are using puli which is beta :smile: